Sources/SwiftCliCore/NestedArrayBuilder.swift

```diff
@@ -8,4 +8,29 @@ public class NestedArrayBuilder {
         let rowStitches = substringRowStitches.map {(String($0))}
         return(rowStitches)
     }
+
+    public func expandRow(row: [String]) -> [String] {
+        let expandedRow = row.flatMap { getMultipleStitchInfo(stitch: $0) }
+        return expandedRow
```

Comment:
Could be just `return row.flatMap...etc.`

Reply:
Done!